### ISSUE TITLE
Can the IPFS cluster group nodes in the cluster, so that the IPFS cluster a region a group, when adding files can specify which groups to add synchronously update content?

### ISSUE THREAD

**Issue by xianyu11111** [ipfs-cluster/ipfs-cluster#2079]
Can the IPFS cluster group nodes in the cluster, so that the IPFS cluster a region a group, when adding files can specify which groups to add synchronously update content?

**Comment by hsanjuan** [comment]
No, but this it worth exploring as an alternative to the --allocations flag. I.e. `--allocations tag:region=eu` or something that limits the set of peers considered to those with certain tags.
